### DISCUSSION TITLE
Trouble with visualizing mesh using Peacock

### DISCUSSION THREAD

**Discussion by leosmith36** [idaholab/moose#21237] [Q&A Tools] [answered]
I am trying to visualize a mesh using Peacock's exodus viewer, and I am able to get the color scale to show but not the image itself. I should be able to see a grid of elemental compositions, and it works in Paraview but not in Peacock. You can see a screenshot of my issue below. I am not sure what else to try, so all advice would be appreciated!

<img width="960" alt="peacock_snip" src="https://user-images.githubusercontent.com/97861087/172651976-e71092e3-3e78-4417-908c-d3eb47d89329.PNG">

My laptop runs Windows 10, and I am running Peacock on Ubuntu 20.04 using the MOOSE conda framework. I open peacock using the `peacock` command in Ubuntu, and then I navigate to the exodus.e file using the exodus viewr. The output I get when I run peacock is:
```
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-<username>'
No executable found for method type(s): opt, dbg, oprof, devel
```

**Comment by GiudGiud** [comment, marked as answer]
Peacock on WSL has issues I think
https://mooseframework.inl.gov/getting_started/installation/windows10.html
so even if you fix this one you ll still have the others. We recommend you work with paraview or with the NEAMS workbench. I ll try to get instructions for the second one and report back

Guillaume

**Reply by GiudGiud** [reply]
best reference for now
https://code.ornl.gov/neams-workbench/downloads
Have not used it on WSL so please let us know if you hit issues there
